fe1-web/parts/lao/socialMedia/SocialSearch.tsx

```diff
@@ -1,11 +1,67 @@
 import * as React from 'react';
-import { Text, View } from 'react-native';
+import {
+  FlatList, ListRenderItemInfo, StyleSheet, View, ViewStyle,
+} from 'react-native';
 import STRINGS from 'res/strings';
+import { makeCurrentLao, makeLastRollCallAttendeesList } from 'store';
+import { useSelector } from 'react-redux';
+import { PublicKey } from 'model/objects';
+import TextBlock from 'components/TextBlock';
+import UserListItem from 'components/UserListItem';
+import { gray } from 'styles/colors';
 
-const SocialSearch = () => (
-  <View>
-    <Text>{STRINGS.social_media_navigation_tab_search}</Text>
-  </View>
-);
+/**
+ * Component that will be used to allow users to search for other users or topics.
+ * For now, it is used to show all the attendees of the last roll call so that everyone can follow
+ * whoever they want.
+ */
+const styles = StyleSheet.create({
+  viewCenter: {
+    alignSelf: 'center',
+    width: 600,
+  } as ViewStyle,
+  titleTextView: {
+    alignSelf: 'flex-start',
+    marginTop: 20,
+  } as ViewStyle,
+  attendeesList: {
+    flexDirection: 'column',
+    marginTop: 20,
+    borderTopWidth: 1,
+    borderColor: gray,
+  } as ViewStyle,
+});
+
+const SocialSearch = () => {
+  const laoSelect = makeCurrentLao();
+  const currentLao = useSelector(laoSelect);
+
+  if (!currentLao) {
+    throw new Error('Impossible to open Social media Search if you are not connected to a LAO');
+  }
+
+  const rollCallId = currentLao.last_tokenized_roll_call_id;
```

Comment:
Do we need to change to rollCallID?

Reply:
I am not changing the roll call id, just retrieving it to be able to make a list of attendees from it (using `makeLastRollCallAttendeesList`).

Reply:
I see, I saw you changed all rollCallIds to rollCallIDs in the previous file. But yes, since it's a const here it makes more sense to name it rollCallId.

Reply:
Oh ok I didn't understand that, sorry. Yeah actually that was because of the name being used in the route. `rollCallId `wasn't recognized in tests because it was not the name of the id in the route (which was `rollCallID`)